### PR TITLE
Add Network to Discord Edgelist

### PR DESCRIPTION
**Pull request by andsv2**
Handles #389: Edgelist lookup for discord_edgelist now looks up relative to indicated network.

### CODE REVIEW

**Comment by smjenness** [comment]
@andsv2 : for development clarity, I will close this PR without merging. Instead, I would like us to first finish the getset branch and PR, merge into _Overhaul, and then work on this discord_edgelist functionality by branching off of the _Overhaul branch. So I'll keep the discord_edgelist branch, but close this PR. Hope that makes sense.

**Comment by andsv2** [comment]
@smjenness #394 ready for review.

File: R/net.mod.infection.R [hunk 1]
```diff
@@ -300,7 +302,7 @@ discord_edgelist <- function(dat, at) {
   if (tergmLite == TRUE) {
     el <- dat$el[[1]]
```
Comment (smjenness) [inline R/net.mod.infection.R:303]: input network here too (there is a parallel structure for the edgelist for multi-network simulations using tergmLite)

Comment (andsv2) [reply R/net.mod.infection.R]: Req. changes updated and passing on Travis.

**Pull request merged**